src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenExprLambdaTests.cs

```diff
@@ -3117,9 +3117,10 @@ public static void Main()
     }
 }";
 
+            // PEVerify: [ : Test::Main][mdToken=0x6000001][offset 0x00000009][found Native Int][expected unmanaged pointer] Unexpected type on the stack.
```

Comment:
One part I'm having trouble deciding on is the comments for the failures. There are effectively three reasons that we have failures:

1. Legitimate failures: consider there are several cases where we run verification with less than the full assembly set. For example missing corelib. Verification fails there essentially by design. 
2. Bugs or limitations in PEVerify: due to the servicing nature of PEVerify these will never be fixed
3. Bugs or limitations in ILVerify: these can be prioritize and fixed

Can't decide if having comments tracking all three cases is worth it. For some reason it just stands out when I'm reviewing the change. Possible that is because this is calling out the ~100 or so times this happens out of the hundreds of thousands of tests that we have hence I'm just looking at the worst case. Means I won't notice in the typical flow of PRs through the repo. 

Don't feel strongly about this but wanted to hear your thoughts a bit.

Reply:
Feels similar to me to leaving a comment when we skip a test. It helps clarify whether we're in situation 1 (test prohibits validation, and clarify that it's not a case where the compiler emitted something bad) or in situations 2 or 3.

Comments for situation 3 are useful to prioritize improving ILVerify (we can do a search for `// ILVerify:` in tests).
I don't much care for comments for situation 2 either (since PEVerify won't get updated), other than clarifying that it's not situation 1.